habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/migration/CustomMonorepoGroupMigration.java

```diff
@@ -0,0 +1,147 @@
+package org.technologybrewery.habushu.migration;
+
+import com.electronwill.nightconfig.core.CommentedConfig;
+import com.electronwill.nightconfig.core.Config;
+import com.electronwill.nightconfig.core.file.FileConfig;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.technologybrewery.baton.AbstractMigration;
+import org.technologybrewery.baton.BatonException;
+import org.technologybrewery.habushu.HabushuException;
+import org.technologybrewery.habushu.util.TomlReplacementTuple;
+import org.technologybrewery.habushu.util.TomlUtils;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Automatically migrates any monorepo dependencies (e.g., foo = {path = "../foo", develop = true}) in the main
+ * [tool.poetry.dependencies] group into the [tool.poetry.group.monorepo.dependencies] group instead.  As noted in the
+ * project's README.md, this prevents these dependencies from causing issues when Poetry projects are exported in
+ * development releases.
+ */
+public class CustomMonorepoGroupMigration extends AbstractMigration {
```

Comment:
This touches on a comment I almost left on the baton PR, but what if someone wants to opt out of this?  Since the migration runs on every build, you couldn't just change it back.  I guess just shut off the migration mojo by setting the phase to none, but then you wouldn't get other/future migrations.

Reply:
Correct - for 0.1.0, that's how they will have to do it.  But in a subsequent change to Baton, we'll let people inactivate migrations in a "line item veto" style.